src/stories/containers/Products/AddProducts/index.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import AddProduct from '../../../../containers/Product/AddProduct';
+import { validators } from '../../../../pages/Cadastros/NovoProduto/ProdutoContainer/validators';
```

Comment:
separar suas importações e as do react,
adicionar uma quebra de linha entre elas 
```suggestion
import React, { useState } from 'react';
import { action } from '@storybook/addon-actions';

import { validators } from '../../../../pages/Cadastros/NovoProduto/ProdutoContainer/validators';
import AddProduct from '../../../../containers/Product/AddProduct';
```